docs/user/filesystem.md

```diff
@@ -740,6 +740,23 @@ When this is done, the local DFuse daemon should shut down the mount point,
 disconnect from the DAOS servers, and exit.  You can also verify that the
 mount point is no longer listed in `/proc/mounts`.
 
+### read() bug in fuse Linux kernel module
+
+We recently observed read() through dfuse returning wrong data occasionally. The
+root cause is a bug in fuse Linux kernel module. The virtual address of user-space
+read buffer was converted to physical page address. Once the data is ready, they
+are copied into the saved physical page address. Under certain circumstances such as
+page migration, the associated physical pages of the virtual address of read buffer
+has been changed. Consequently, fuse writes expected data into a wrong place and
+the read buffer holds wrong data. We speculate that such issue might be triggered
+by bad memory fragmentation. The issue has been reproduced under various versions
+of kernels, ranging from 3.10.0 to 6.3.6. Older kernels are not tested. We have
+a working patch to fix this bug under Linux kernel 6.3.6. This patch pins physical
```

Comment:
This is not a patch.

Reply:
What do you mean "this" here? Thank you!

Reply:
The line I commented on.

Overall this reads like a commit message or a change-log entry for the kernel, it's not something that's applicable as-is to users of dfuse.

Reply:
ok. I can remove some details. Thank you!